sql-statements/sql-statement-create-index.md

```diff
@@ -10,37 +10,38 @@ aliases: ['/docs-cn/v2.1/sql-statements/sql-statement-create-index/','/docs-cn/v
 
 ## 语法图
 
-**CreateIndexStmt:**
+```ebnf+diagram
+CreateIndexStmt ::=
+    'CREATE' IndexKeyTypeOpt 'INDEX' IfNotExists Identifier IndexTypeOpt 'ON' TableName '(' IndexPartSpecificationList ')' IndexOptionList IndexLockAndAlgorithmOpt
 
-![CreateIndexStmt](/media/sqlgram/CreateIndexStmt.png)
+CreateIndexStmtUnique ::=
+    UNIQUE?
 
-**CreateIndexStmtUnique:**
+Identifier ::=
+    identifier
+|   UnReservedKeyword
+|   NotKeywordToken
+|   TiDBKeyword
 
-![CreateIndexStmtUnique](/media/sqlgram/CreateIndexStmtUnique.png)
+IndexTypeOpt ::=
+    IndexType?
 
-**Identifier:**
+TableName ::=
+    Identifier ('.' Identifier)?
 
-![Identifier](/media/sqlgram/Identifier.png)
+IndexColNameList ::=
+    IndexColName (',' IndexColName)*
```

Comment:
Same as the previous comment.

Reply:
superseded by `IndexPartSpecificationList`.

Reply:
Is it alright to use `IndexPartSpecificationList` in docs for release 2.1?

Reply:
if `CREATE INDEX i ON t (col(123));` works on 2.1 then it should be IndexPartSpecificationList.

if not the `IndexPartSpecificationList` in `CreateIndexStmt` should be replaced by `IndexColNameList`.